### ISSUE TITLE
Welcome!

### ISSUE THREAD

**Issue by SolidusDigital** [SolidusDigital/invest-and-chill#1]
Please post your feed back here as a new issue. 

Any feedback is welcome. - problems - bugs - suggestions, etc. 

Also when the contest starts we will update this issue. Currently we are waiting on some more traders. 
PS. Git hub allows you to copy paste images.
